Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -1,39 +1,50 @@
 ﻿using JokesOnYou.Web.Api.DTOs;
 using JokesOnYou.Web.Api.Models;
+using JokesOnYou.Web.Api.Repositories;
 using JokesOnYou.Web.Api.Repositories.Interfaces;
 using JokesOnYou.Web.Api.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Components;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace JokesOnYou.Web.Api.Services
 {
+    
     public class UserService : IUserService
     {
-        private readonly IUserRepository _userRepository;
-
-        public UserService(IUserRepository userRepository)
+        readonly IUserRepository _userRepo;
+        public  UserService(IUserRepository userRepo)
```

Comment:
UserService needs to do all the work but is currently not doing ANY work. Though this is not your Job/Issue/Work ;)
Though we should change our workflow like we are currently discussing on Discord.